code/modules/uplink/uplink_item.dm

```diff
@@ -965,6 +886,92 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 4
 	include_modes = list(/datum/game_mode/nuclear)
 
+// Space Suits & Hardsuits
+/datum/uplink_item/space_suits
+	category = "Space Suits & Hardsuits"
+
+/datum/uplink_item/space_suits/space_suit
+	name = "Syndicate Space Suit"
+	desc = "This red and black syndicate space suit is less encumbering than Nanotrasen variants, \
+			fits inside bags, and has a weapon slot. Nanotrasen crewmembers are trained to report red space suit \
+			sightings, however."
+	item = /obj/item/weapon/storage/box/syndie_kit/space
+	cost = 4
+	exclude_modes = list(/datum/game_mode/gang)
+
+/datum/uplink_item/space_suits/hardsuit
+	name = "Syndicate Hardsuit"
+	desc = "The feared suit of a syndicate nuclear agent. Features slightly better armoring and a built in jetpack \
+			that runs off standard atmospheric tanks. When the built in helmet is deployed your identity will be \
+			protected, even in death, as the suit cannot be removed by outside forces. Toggling the suit in and out of \
+			combat mode will allow you all the mobility of a loose fitting uniform without sacrificing armoring. \
+			Additionally the suit is collapsible, making it small enough to fit within a backpack. \
+			Nanotrasen crew who spot these suits are known to panic."
+	item = /obj/item/clothing/suit/space/hardsuit/syndi
+	cost = 8
+	include_modes = list(/datum/game_mode/nuclear)
```

Comment:
traitors buying nuke op hardsuits is a feature and a metabreaker, and you stealth removing it because you personally don't like it is really shitty
also, nuke op hardsuit != elite hardsuit